Dockerfile

```diff
@@ -299,8 +299,9 @@ RUN \
     -DBUILD_STATIC_LIBS=ON \
     -DCMAKE_INSTALL_PREFIX=/usr/local \
     -DCMAKE_INSTALL_LIBDIR=lib \
+    -DCMAKE_BUILD_TYPE=Release \
     .. \
-  && cmake --build . --config Release --target install
+  && cmake --build --target install .
```

Comment:
Build fail with `#24 1.031 Unknown argument install` wonder if `--build` still wants the "."?

Reply:
i might have messed up here with this. it seems obvious now. As this is absolutely not important, i will close

Reply:
no worries! will probably cleanup the cmake stuff at some point if you don't beat me to it